ajaxorg/ace
# AsciiDoc demo

While the `ace-builds` repository (which is meant, I take it, to contain generated files) contains a `kitchen-sink/docs/AsciiDoc.asciidoc` file, that file doesn't get generated from the current master branch.

As it turns out, the file name `AsciiDoc.asciidoc` (note the uppercase) broke the kitchen sink, so I put it in lowercase, which works.

Detail of how to reproduce the breakage:
1. Go to http://ace.c9.io/build/kitchen-sink.html
2. Click on the Document widget, select AsciiDoc.
3. Notice that the page that appears is the HTML code for a GitHub 404.

I am unsure whether this pull request can cleanly be accepted, but in case it can, I have signed the CLA independently.
